notebooks/official/pipelines/google_cloud_pipelines_dataproc_tabular/google_cloud_pipeline_components_dataproc_tabular.ipynb

```diff
@@ -32,21 +32,22 @@
         "<table align=\"left\">\n",
```

Comment:
<p>Do not use 1st person (we, us, let's). </p><p> </p><p>Vertex AI Metadata -&gt; Vertex ML Metadata</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/711/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='70'/>

Reply:
<p>done</p>

Reply:
done